### ISSUE TITLE
Any simple way to get price or symbol while iterating positions?

### ISSUE THREAD

**Issue by doctorcolossus** [sshh12/Tradinhood#2]
Maybe I'm missing this feature, but I can't see either of those attributes on positions in the API.
My workaround for now is:
```
for position in rbh.positions:
  print(rbh[rbh.session.get(position['instrument']).json()['symbol']].price)
```
If the API supports a simpler approach, please document it.
Otherwise, please consider adding this functionality.

Tradinhood rules. Thank you.

**Comment by sshh12** [comment]
Hi!

As I develop the library I plan on switching out the JSON returns to Tradinhood Objects, like how ```rbh.orders``` now returns a list of orders rather than the raw JSON. Right now both ```rbh.positions``` and ```rbh.holdings``` will return JSONs.

**tl;dr** Not yet. Soon.

**Comment by sshh12** [comment]
```python
>>> rbh.get_assets()
{<Stock (Inpixon) [INPX]>: Decimal('0.0000'), <Stock (Yext) [YEXT]>: Decimal('0.0000'), <Stock (Micron Technology) [MU]>: Decimal('5.0000'), <Stock (Intel) [INTC]>: Decimal('0.0000'), <Stock (Ford) [F]>: Decimal('0.0000'), <Stock (AMD) [AMD]>: Decimal('0.0000'), <Currency (Dogecoin) [DOGE]>: Decimal('152.000000000000000000'), <Currency (Bitcoin) [BTC]>: Decimal('0E-18'), <Currency (Ethereum) [ETH]>: Decimal('0E-18')}

>>> rbh.get_assets()[rbh['DOGE']]
Decimal('152.000000000000000000')

>>> {asset.symbol: asset.price for asset in rbh.get_assets().keys()}
{'INPX': Decimal('0.133700'), 'YEXT': Decimal('24.396400'), 'MU': Decimal('48.520000'), 'INTC': Decimal('46.540000'), 'F': Decimal('9.715000'), 'AMD': Decimal('20.036900'), 'DOGE-USD': Decimal('0.002426'), 'BTC-USD': Decimal('6459.585000'), 'ETH-USD': Decimal('288.935000')}
```

**Comment by doctorcolossus** [comment]
Awesome, thank you. :)
